src/etc/inc/services.inc

```diff
@@ -1153,9 +1155,10 @@ EOD;
 
 				if (isset($sm['ddnsupdate'])) {
 					if (($sm['ddnsdomain'] <> "") && ($sm['ddnsdomain'] != $dhcpifconf['ddnsdomain'])) {
-						$smdnscfg .= "		ddns-domainname \"{$sm['ddnsdomain']}\";\n";
+						$smdnscfg .= "        ddns-domainname \"{$sm['ddnsdomain']}\";\n";
+				 		$need_sm_ddns_updates = true;	
 					}
-					$smdnscfg .= "		ddns-update-style interim;\n";
+					$smdnscfg .= "        ddns-update-style interim;\n";
```

Comment:
Why change from tabs to spaces? The rest of the file uses tabs.

Reply:
I changed it because the resulting output was indented differently than the rest. With that change all configuration options are in line.